scripts/db/schema.sql

```diff
@@ -170,7 +170,7 @@ ALTER TABLE ONLY evaluations
 
 CREATE TABLE users (
     id integer NOT NULL,
-    csid character varying(255) NOT NULL,
```

Comment:
There are a few dept contacts listed in the source spreadsheets (from Summer 2021) who do not have a CSID in CalNet.